web/yo/app/scripts/directives/driverealtimestring.js

```diff
@@ -64,7 +64,7 @@ angular.module('oncokbApp')
                         if(scope.rs && _.isNull(scope.rs.get('lastReviewed')) && scope.rs.get('action') !== 'rejected') {
                             scope.rs.set('lastReviewed', o);
                         }
-                        scope.content.stringO = stringUtils.getTextString(scope.object.text);
+                        scope.content.stringO = scope.object.text;
```

Comment:
Could you take a look this issue ?https://github.com/oncokb/oncokb/commit/3d4f3decb3cbf3f25318d7bf9e72e4ee1162ff70